src/sentry/runner/commands/devserver.py

```diff
@@ -52,6 +52,18 @@
         "--synchronize-commit-group=transactions_group",
         "--no-strict-offset-reset",
     ],
+    "post-process-forwarder-issue-platform": [
```

Comment:
Yup, good callout. Was planning to wait till we sync with OPS before merging this in to avoid any incidents.

Reply:
Hmm, was gonna create a new PR to revert the revert that added the commit log to prod settings, but that looks like it hasn't merged https://github.com/getsentry/sentry/pull/45014

Not sure what's the status of prod snuba but won't be merging this until snuba is in a known state.

Reply:
@lynnagara I think we should be good to deploy this based on this comment from Ben
https://github.com/getsentry/ops/pull/6180#issuecomment-1442180107

Reply:
Or maybe you're referring to this PR which should start producing to the topic https://github.com/getsentry/snuba/pull/3775/files.

Or maybe we should merge this first which consumes from the commit log topic before producing?

Reply:
Yep. https://github.com/getsentry/snuba/pull/3775](https://github.com/getsentry/snuba/pull/3775/

We should not merge this first or it may break devserver